Implemented new docs structure (#53)
dappforce/subsocial-docs: docs/develop/how-to-guides/Structures.md

---
id: structures
title: Blockchain Structures
displayed_sidebar: developSidebar
---

> **This section covers how spaces, posts and reactions are related to each other.**

Reviewer: ```suggestion
**This section covers how spaces, posts, and reactions are related to each other.**
```

Subsocial uses a few building blocks to enable social networking. 

> A quick glimpse of how the major components i.e. Space, Post and Reactions are related.

Reviewer: ```suggestion
Here we will take a quick glimpse of how the major components such as Spaces, Posts, and Reactions are related.
```
